compute/api/Compute.Samples.Tests/ComputeFixture.cs

```diff
@@ -52,6 +54,9 @@ public class ComputeFixture : IDisposable, ICollectionFixture<ComputeFixture>
 
         private StorageClient StorageClient { get; } = StorageClient.Create();
 
+        public RetryRobot Assert { get; } = new RetryRobot { RetryWhenExceptions = new Type[] { typeof(XunitException) } };
+
```

Comment:
If you feel like removing one of these blank lines, that would be good - but don't worry :)

Reply:
Done!